test_unstructured_ingest/expected-structured-output/s3/recalibrating-risk-report.pdf.json

```diff
@@ -660,9 +704,383 @@
     }
   },
   {
-    "type": "NarrativeText",
-    "element_id": "26963be98ae7ff2e8c0428862d074cf6",
-    "text": "r e p s e i t i l",
+    "type": "Title",
+    "element_id": "d78a11e9e55235934c3a4922053c68e5",
+    "text": "e",
+    "metadata": {
+      "filetype": "application/pdf",
+      "languages": [
+        "eng"
+      ],
+      "page_number": 5,
+      "data_source": {
+        "url": "s3://utic-dev-tech-fixtures/small-pdf-set/recalibrating-risk-report.pdf",
+        "version": "e690f37ef36368a509d150f373a0bbe0",
+        "record_locator": {
+          "protocol": "s3",
+          "remote_file_path": "s3://utic-dev-tech-fixtures/small-pdf-set/"
+        },
+        "date_created": "1676196572.0",
+        "date_modified": "1676196572.0"
+      }
+    }
+  },
+  {
+    "type": "Title",
+    "element_id": "8d14df8b7fd7744365fbf8e02d69415a",
+    "text": "y",
+    "metadata": {
+      "filetype": "application/pdf",
+      "languages": [
+        "eng"
+      ],
+      "page_number": 5,
+      "data_source": {
+        "url": "s3://utic-dev-tech-fixtures/small-pdf-set/recalibrating-risk-report.pdf",
+        "version": "e690f37ef36368a509d150f373a0bbe0",
+        "record_locator": {
+          "protocol": "s3",
+          "remote_file_path": "s3://utic-dev-tech-fixtures/small-pdf-set/"
+        },
+        "date_created": "1676196572.0",
+        "date_modified": "1676196572.0"
+      }
+    }
+  },
+  {
+    "type": "Title",
+    "element_id": "f4df01bee1b8ffb973ac8539649c5189",
+    "text": "W",
```

Comment:
just a note that those vertical text are now broken into each letter per element; not sure what the impact would be but just a note